frontend/src/pages/AgencyProfileForm/DistributionCalendar/CalendarStyle.css

```diff
@@ -0,0 +1,154 @@
+.calendar-container {
```

Comment:
Can you add some inline comments to this file just to break up the different styling and identify larger components

Reply:
Seconding this, maybe just add some comments indicating purpose of the styling